packages/patternfly-3/patternfly-react/sass/patternfly-react/_dual-list-selector.scss

```diff
@@ -0,0 +1,127 @@
+.pf-dual-list-selector {
+  display: inline-block;
+  width: 230px;
+  min-width: 230px;
```

Comment:
Absolute numbers like this width and min-width should be assigned to variables, especially when they coordinate with other styles - looks like this matches the size of `.pf-dual-list-body` so both should use the same variable if that's true.